Sources/MapboxNavigation/Resources/de.lproj/Localizable.strings

```diff
@@ -91,6 +94,9 @@
 /* Label indicating the device volume is too low to hear spoken instructions and needs to be manually increased */
 "INAUDIBLE_INSTRUCTIONS_CTA" = "Lautstärke anpassen, um Anweisungen zu hören";
 
+/* Specific route feedback that a speed limit is incorrect. */
+"INCORRECT_SPEED_LIMIT" = "Speed limit incorrect";
```

Comment:
Why are there so many unrelated changes?

Reply:
These changes are not intented for this PR, should be updated, thanks!